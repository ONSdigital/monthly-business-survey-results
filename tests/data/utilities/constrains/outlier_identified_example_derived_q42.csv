reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value
1,202001,16,40,100,1,100
1,202001,16,42,100,1,100
