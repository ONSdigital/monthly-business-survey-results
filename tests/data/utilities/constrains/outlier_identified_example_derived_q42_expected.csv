reference,period,spp_form_id,questioncode,value,outlier_weight,winsorised_value,post_winsorised
1,202001,16,40,100,1,100,FALSE
1,202001,16,42,100,1,100,FALSE
